_sass/catalog.scss

```diff
@@ -760,6 +759,8 @@ label {
 
 .pre-white-space {
   white-space: pre-wrap;
+  overflow-y: scroll;
```

Comment:
no scroll needed

Reply:
Hello. Apologies for the delay in communication. I have re-adjusted the overflow behavior as requested and have applied the required changes. 
Would request a re-review of the PR. 
Thanks.